src/frontend/Docs/Entry.elm

```diff
@@ -83,6 +83,34 @@ tagMap func tag =
 
 
 
+-- FILTER
+
+
+nameDistance : String -> Model Type -> Int
+nameDistance query model =
+  case model.info of
+    Value tipe _ ->
+      if query == model.name then
+        0
+      else if String.contains query model.name then
+        1
+      else
+        10
+
+    _ ->
+      100
```

Comment:
Yep. Those have to match the ones in `Docs.Type`, missing there as constants as well at the moment. I'm not really sure how many different values we need and how to name them. Needs some exploration.